clink/lua/src/settings_api.cpp

```diff
@@ -153,11 +153,11 @@ template <typename S, typename... V> void add_impl(lua_State* state, V... value)
 /// <span class="arg">default</span> when the setting isn't explicitly set.
 ///
 /// The type of <span class="arg">default</span> determines what kind of setting
-/// is added:  boolean, integer, and string values add the corresponding setting
-/// type.  Or if the type is table then an enum setting is added:  the table
+/// is added:  boolean and string values add the corresponding setting type.
+/// Or if the type is table then an enum setting is added:  the table
```

Comment:
The original text is accurate.
How was it determined that integers aren't supported?
```lua
settings.add("name", 100) -- This adds an integer setting called "name", with a default value of 100.
```
If you set a breakpoint on line 185, you should see it get hit.